run_create_s3_srr.py

```diff
@@ -119,7 +135,11 @@ def run_create_s3_srr_bucket(args):
                 "DeleteMarkerReplication": {"Status": "Disabled"},
                 "Filter": {"Prefix": ""},
                 "Destination": {
-                    "Bucket": "arn:aws:s3:::%s" % replication_bucket_name
+                    "Bucket": "arn:aws:s3:::%s" % replication_bucket_name,
```

Comment:
@GoldieKMG // 근래에 string format 을 f-string 으로 고치고 있는데 아래와 같이 수정 하시면 좋을 것 같습니다.
```suggestion
                    "Bucket": f"arn:aws:s3:::{replication_bucket_name}",
```